integration_tests/snaps/qbs-nil/snapcraft.yaml

```diff
@@ -0,0 +1,16 @@
+name: test-qbs-plugin
+version: 0.1
+summary: qbs snapcraft plugin
+description: Tests if parts can be built using qbs and gcc
+confinement: strict
+
+parts:
+  qbs-project:
+    plugin: qbs
+    source: .
+    qbs-options: [
+      'project.printString:true'
+    ]
```

Comment:
nit comment, so far I haven't seen this style in yaml.
I've seen:

qbs-options: ['project.printString:true']

and 

qbs-options:
  - 'project.printString:true'

Have you seen that usage in other places? If not, please change it.